app/build.gradle

```diff
@@ -11,10 +11,10 @@ ext {
 dependencies {
   implementation project(':persistence:model')
 
-  implementation project(':domain-rrd:rrd-shared')
 
   implementation project(':domain-xample:xample-api-controller')
   implementation project(':domain-cc:cc-api-controller')
+  implementation project(':svc-bip-api')
```

Comment:
Please add a code comment to state that this is temporary. cc: @greene858 @engineer-plus-plus